components/brave_wallet_ui/utils/network-utils.ts

```diff
@@ -110,3 +110,10 @@ export const hasEIP1559Support = (
     network.isEip1559
   )
 }
+
+export const reduceNetworkDisplayName = (name?: string) => {
+  if (!name) {
+    return ''
+  }
+  return name.split(' ')[0]
+}
```

Comment:
cc: @Douglashdaniel, who added this. I believe the Send-like UX fixes this?

The use of `reduceNetworkDisplayName` is less of a security issue here because this is not the signing screen in the panel.